todos-dynamodb/Sources/App/Controllers/TodoController.swift

```diff
@@ -71,7 +71,7 @@ struct TodoController: Sendable {
     }
 
     @Sendable func updateId(_ request: HBRequest, context: some HBRequestContext) async throws -> Todo {
-        guard var todo = try? await request.decode(as: EditTodo.self, context: context) else { throw HBHTTPError(.badRequest) }
+        var todo = try await request.decode(as: EditTodo.self, context: context)
         guard let id = context.parameters.get("id", as: UUID.self) else { throw HBHTTPError(.badRequest) }
```

Comment:
I like `require` more than `get` in these cases, what do you think?

Reply:
yep fixed all the ones in this file